make some comments in cd.md file

duochen97 opened the pull request:
# Advanced HW 5 submission

Closes #[your issue claim number]

Name: [your name]
Uniqname: [your uniqname]

AmritHariharan commented on _commands/basics/cd.md:
```diff
@@ -85,4 +85,4 @@ Alborz-Brals-MacBook-Pro:basics abral$ pwd
 /
-~~~
\ No newline at end of file
+~~~
```
no actual documentation for the command

duochen97 commented:
Hi Amrit, sorry for adding some non-trivial commits, I have added some
documentation for command.

On Sun, Feb 18, 2018 at 1:05 AM, Amrit Hariharan <notifications@github.com>
wrote:

> *@AmritHariharan* requested changes on this pull request.
>
> you need to give a few examples of how the command would be used and
> explain what they do. Look at the chmod command on the site for an
> example of what we expect.
> ------------------------------
>
> In _commands/basics/cd.md
> <https://github.com/c4cs/c4cs.github.io/pull/510#discussion_r168926952>:
>
> > \ No newline at end of file
> +~~~
>
> no actual documentation for the command
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/c4cs/c4cs.github.io/pull/510#pullrequestreview-97379379>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AgM3NVSNieS36Gxv4ftD-VskP38QCJLoks5tVwbCgaJpZM4SBBIn>
> .
>

duochen97 commented:
Hi Amrit,

      I have added documentation for command cd, can I get points for
advance hw?

Best,
Duo Chen

On Sun, Feb 18, 2018 at 1:30 AM, Duo Chen <duochen@umich.edu> wrote:

> Hi Amrit, sorry for adding some non-trivial commits, I have added some
> documentation for command.
>
> On Sun, Feb 18, 2018 at 1:05 AM, Amrit Hariharan <notifications@github.com
> > wrote:
>
>> *@AmritHariharan* requested changes on this pull request.
>>
>> you need to give a few examples of how the command would be used and
>> explain what they do. Look at the chmod command on the site for an
>> example of what we expect.
>> ------------------------------
>>
>> In _commands/basics/cd.md
>> <https://github.com/c4cs/c4cs.github.io/pull/510#discussion_r168926952>:
>>
>> > \ No newline at end of file
>> +~~~
>>
>> no actual documentation for the command
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/c4cs/c4cs.github.io/pull/510#pullrequestreview-97379379>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AgM3NVSNieS36Gxv4ftD-VskP38QCJLoks5tVwbCgaJpZM4SBBIn>
>> .
>>
>
>

AmritHariharan commented:
You need to have a decent amount more than this, look at the documentation for the `uniq` command on the site for an example of what we expect for the documentation for a command.
